kickstart/etc/dnsmasq-default

```diff
@@ -1,28 +0,0 @@
-#DOMAIN_SUFFIX=`dnsdomainname`
-
-# Select an alternative config file by setting DNSMASQ_OPTS
-# to --conf-file=<file>
-#DNSMASQ_OPTS="--conf-file=/etc/dnsmasq.alt"
-
-# Whether or not to run the dnsmasq daemon; set to 0 to disable.
-ENABLED=1
-
-# By default search this drop directory for configuration options.
-# Libvirt leaves a file here to make the system dnsmasq play nice.
-# Comment out this line if you don't want this. The dpkg-* are file
-# endings which cause dnsmasq to skip that file. This avoids pulling
-# in backups made by dpkg.
-CONFIG_DIR=/etc/dnsmasq.d,.dpkg-dist,.dpkg-old,.dpkg-new
-
-# If the resolvconf package is installed, dnsmasq will use its output
-# rather than the contents of /etc/resolv.conf to find upstream
-# nameservers. Uncommenting this line inhibits this behaviour.
-# Note that including a "resolv-file=<filename>" line in
-# /etc/dnsmasq.conf is not enough to override resolvconf if it is
-# installed: the line below must be uncommented.
-#IGNORE_RESOLVCONF=yes
-
-# Use DNSMASQ_EXCEPT to prevent dnsmasq from listening on the listed
-# interfaces.  Additionally, include "lo" to keep from adding 127.0.0.1
-# as a local DNS server via resolvconf.
-DNSMASQ_EXCEPT="lo {{posm_wan_netif}}"
```

Comment:
We actually do want this file, but only with `DNSMASQ_EXCEPT="lo"`, as `/etc/init.d/dnsmasq` uses that to determine whether to update `/etc/resolv.conf` (i.e. for local DNS).

A side-effect of this is that DNS lookups for `posm.io` from the POSM itself will use `/etc/hosts` (or public DNS, if absent) and therefore resolve to either `127.0.1.1` or `export.posm.io` (currently). This may be ok, provided that relevant services (Nginx, etc.) are listening on `localhost` (which they should be by default).